sdk/src/main/java/io/dapr/client/domain/CloudEvent.java

```diff
@@ -255,6 +371,104 @@ public void setBinaryData(byte[] binaryData) {
     this.binaryData = binaryData == null ? null : Arrays.copyOf(binaryData, binaryData.length);
   }
 
+  /**
+   * Gets the pubsub component name.
+   * @return the pubsub component name.
+   */
+  public String getPubsubname() {
+    return pubsubname;
+  }
+
+  /**
+   * Sets the pubsub component name.
+   * @param pubsubname the pubsub component name.
+   */
+  public void setPubsubname(String pubsubname) {
+    this.pubsubname = pubsubname;
+  }
+
+  /**
+   * Gets the topic name.
+   * @return the topic name.
+   */
+  public String getTopic() {
+    return topic;
+  }
+
+  /**
+   * Sets the topic name.
+   * @param topic the topic name.
+   */
+  public void setTopic(String topic) {
+    this.topic = topic;
+  }
+
+  /**
+   * Gets the time.
+   * @return the time.
+   */
+  public OffsetDateTime getTime() {
+    return time;
+  }
+
+  /**
+   * Sets the time.
+   * @param time the time.
+   */
+  public void setTime(OffsetDateTime time) {
+    this.time = time;
+  }
+
+  /**
+   * Gets the trace id which is the legacy name for trace parent.
+   * @return the trace id.
+   */
+  @Deprecated
+  public String getTraceid() {
+    return traceid;
+  }
+
+  /**
+   * Sets the trace id which is the legacy name for trace parent.
+   * @param traceid the trace id.
+   */
+  @Deprecated
+  public void setTraceid(String traceid) {

```

Comment:
camelCase in every setter and getter too: setTraceId and getTraceId, for example.